flask/packages.py

```diff
@@ -33,6 +34,24 @@ def search_packages():
     )
     page = int(page) if page else 0
 
+    packages_per_page = 10
+
+    # Count the number of documents in the database related to query.
+    total_documents = db.packages.count_documents({
```

Comment:
can you shift this down after the query has been formatted. for similar outputs to the queries.